internal/dcache/replication_manager/replication_manager.go

```diff
@@ -65,6 +66,11 @@ type replicationMgr struct {
 
 var rm *replicationMgr
 
+// Set of currently running sync jobs, indexed by target replica ("rvX/mvY") and the value stored is the
+// source replica in "rvX/mvY" format.
+// Note that there can only be a single sync job for a given target replica.
+var runningJobs sync.Map
```

Comment:
We can move this inside the `replicationMgr` struct. There is already a global variable for `replicationMgr`